microsoft-edge/devtools-guide-chromium/progressive-web-apps/simulate-window-controls-overlay.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Simulate PWA Window Controls Overlay
+description: Simulate PWA Window Controls Overlay.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 05/13/2024
+---
+# Simulate PWA Window Controls Overlay
+
+In DevTools, in the **Application** tool, you can simulate the Window Controls Overlay feature that Progressive Web Apps (PWAs) have, without having to install the PWA.  The **Application** tool has a **Window Controls Overlay** section under **Manifest**, to display a preview of how your Progressive Web App (PWA) handles window controls, without having to install the PWA and make the styling changes that are necessary to accommodate window controls:
```

Comment:
```suggestion
In DevTools, in the **Application** tool, you can simulate the Window Controls Overlay (WCO) feature that Progressive Web Apps (PWAs) have without having to install the PWA first and enable the WCO feature. By simulating the WCO feature from DevTools, you can test your title bar area CSS code changes more quickly.
```

Feel free to rephrase this, but it felt a little more intuitive to me like this.